c2corg_ui/templates/helpers/list.html

```diff
@@ -4,6 +4,7 @@
     <option translate value="${request.route_url('waypoints_index')}" ng-selected="{{mainCtrl.isPath('waypoints')}}">Waypoints</option>
     <option translate value="${request.route_url('outings_index')}" ng-selected="{{mainCtrl.isPath('outings')}}">Outings</option>
     <option translate value="${request.route_url('images_index')}" ng-selected="{{mainCtrl.isPath('images')}}">Images</option>
+    <option translate value="${request.route_url('areas_index')}" ng-selected="{{mainCtrl.isPath('areas')}}">Areas</option>
   </select>
```

Comment:
you should also update the `mainCtrl.isPath()` function - it will highlight "topoguide" in the sidemenu.